src/FsSpreadsheet.ExcelIO/Cell.fs

```diff
@@ -137,26 +137,21 @@ module Cell =
         | _  -> 
             let valType,value = inferCellValue value
             let reference = CellReference.ofIndices columnIndex (rowIndex)
-            create valType reference (CellValue.create value)
+            create (Some valType) reference (CellValue.create value)
             |> fun c ->
                 if value.EndsWith " " then
                     setSpacePreserveAttribute c
                 else c
 
-    /// <summary>
-    /// Create a cell using a shared string table, also returns the updated shared string table.
-    /// </summary>
-    let fromValueWithDataType (sharedStringTable : SharedStringTable Option) columnIndex rowIndex (value : string) (dataType : DataType) = 
+    let getCellContent (doc : Packaging.SpreadsheetDocument) (value : string) (dataType : DataType) = 
```

Comment:
I don't like this name anymore.. 
Any idea for a better one?

Reply:
Which function name do you specifically mean by this?

Reply:
`getCellContent`
It's not really clear whether it's about getting the content of an FsCell or an OpenDocument Cell.